Backend/Remora.Discord.Rest/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -68,6 +68,28 @@ public static IServiceCollection AddDiscordRest
         Func<IServiceProvider, (string Token, DiscordTokenType TokenType)> tokenFactory,
         Action<IHttpClientBuilder>? buildClient = null
     )
+    {
+        serviceCollection.AddSingleton<IAsyncTokenStore>(
```

Comment:
Formatting.

Reply:
Which would be the preferred format in this case?
```cs
        serviceCollection.AddSingleton<IAsyncTokenStore>(ctx =>
        {
            var (token, type) = tokenFactory(ctx);
            return new StaticTokenStore(token, type);
        });
````
or
```cs
        serviceCollection.AddSingleton<IAsyncTokenStore>
        (
            ctx =>
            {
                var (token, type) = tokenFactory(ctx);
                return new StaticTokenStore(token, type);
            }
        );
```

Reply:
The former is fine in single-argument contexts 🙂